index.js

```diff
@@ -5,5 +5,8 @@ module.exports = {
   PaymentMedium: require('./src/payment-medium'),
   Quote: require('./src/quote'),
   Trade: require('./src/trade'),
-  Helpers: require('./src/helpers')
+  Helpers: require('./src/helpers'),
+  mocks: {
```

Comment:
That said, I'm not sure if this is the best way to create such a mock. This mock is a minimal recreation of the original class. This makes sense for unit tests _within_ the `bitcoin-exchange-client` project, it's important to only test one class at a time.

However for other projects that use this project, it's more important to have a complete representation. A better approach for that might be to mock only the API class.

I'll get rid of this export for now.